decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -52,6 +54,14 @@ def permissions
 
       private
 
+      def view_private_users_menu?
+        return unless permission_action.subject == :space_private_user
+
```

Comment:
```suggestion
```

Let's keep the guard clauses together

Reply:
fixed in [bb74aa7](https://github.com/decidim/decidim/pull/12031/commits/bb74aa7c35909f58337992441e4040338a5c8537)